lib/libutee/include/elf_common.h

```diff
@@ -683,6 +684,13 @@ typedef struct {
 #define	R_386_TLS_TPOFF32	37	/* GOT entry of -ve static TLS offset */
 #define	R_386_IRELATIVE		42	/* PLT entry resolved indirectly at runtime */
 
+#define	R_RISCV_NONE		0	/* No relocation. */
+#define	R_RISCV_64			2	/* 64-bit relocation. */
+#define	R_RISCV_RELATIVE	3	/* Adjust a link address (A) to its
+									load address (B + A). */
```

Comment:
Please align with the start of "Adjust" on the line above. Same comment two lines down.